bin/tests

```diff
@@ -29,6 +29,12 @@ if [ $ATTEMPTS -eq 5 ]; then
   exit 1
 fi
 
+if [ -n "${OPEN_SHELL:-}" ]; then
+    export PYTHONPATH=/opt/warehouse/src/
+    echo "Welcome to the test shell. Use 'pytest' to run the tests."
```

Comment:
Note: spaces are important. `"Welcome to the test shell" != "Welcome to the tests hell"`.

❤️